static/js/compose.js

```diff
@@ -623,7 +584,6 @@ exports.respond_to_message = function (opts) {
     var msg_type;
     // Before initiating a reply to a message, if there's an
     // in-progress composition, snapshot it.
-    compose.snapshot_message();
```

Comment:
To reply to a message, you need to click outside of the compose box right or on some other message? That would trigger the focusout handler.